app/helpers/money_formatting_helper.rb

```diff
@@ -0,0 +1,21 @@
+module MoneyFormattingHelper
+  def format_money_input(log:, question:)
+    value = log[question.id]
+
+    return unless value
+    return value unless question.prefix == "£"
+
+    number_with_precision(
+      value,
+      precision: 2,
+    )
+  end
+
+  def format_as_currency(num_string)
+    ActionController::Base.helpers.number_to_currency(
```

Comment:
if this is in a helpers folder, does it definitely need the ActionController::Base.helpers prefix?